blog/poc3.md

```diff
@@ -47,13 +45,16 @@ MRU is an experimental feature in current POC3 testnet and is still undergoing c
 ## FUSE support
 
 [FUSE](http://swarm-guide.readthedocs.io/en/latest/usage.html#FUSE), enables users to integrate Swarm data directly into their local file systems (only on Linux and Mac). Using this system, users can "mount a Swarm manifest" as if it were a regular directory. It supports file system read and write operations, in which all content is automatically synced with the Swarm.
-In future, combining FUSE with Swarm Mutable Resources, it should be possible sync your entire home folder between devices - the backend to a decentralised storage with dropbox-like functionality.
+In future, combining FUSE with Swarm Mutable Resources, it should be possible, for example, to sync your entire home folder between devices - the backend to a decentralised storage with Dropbox-like functionality.
 
+## Encryption support
+
+Swarm 0.3 comes with built-in [*encryption*](http://swarm-guide.readthedocs.io/en/latest/usage.html#Encryption) allowing for secure uploads of private data. With the planned implementation of *Access Control Trees*, we will have the ability to upload a directory privately and still 'share' a subdirectory with specific peers.
```

Comment:
well that is true but that is already possible. so I suggest:

allowing for secure uploads of private data. The way encryption works users can pload a directory privately and still 'share' a subdirectory with specific peers. 
*Access Control Trees* (Swarm 0.3.2) will offer an API for users to manage access to assets independantly of publishing content. Granted access will work across versions of a mutable resources.